health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/MultiSelectDropdown.js

```
@@ -135,7 +136,7 @@ const Wrapper = ({ boundaryOptions, setShowPopUp, alreadyQueuedSelectedState, on
     >
       <LabelFieldPair>
         <CardLabel>
-        {t((hierarchyType + "_" + boundaryType).toUpperCase())}
+          {t((hierarchyType + "_" + boundaryType).toUpperCase())}
```

Biome

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)